web/packages/teleport/src/Roles/RoleEditor/StandardEditor/standardmodel.ts

```diff
@@ -1092,9 +1098,12 @@ export function roleEditorModelToRole(roleModel: RoleEditorModel): Role {
         break;
 
       case 'git_server':
-        role.spec.allow.github_permissions = [
-          { orgs: optionsToStrings(res.organizations) },
-        ];
+        const orgs = optionsToStrings(res.organizations);
+        if (orgs.length > 0) {
+          role.spec.allow.github_permissions = [
+            { orgs: optionsToStrings(res.organizations) },
```

Comment:
We have `orgs` above.
```suggestion
            { orgs },
```